### ISSUE TITLE
Turf test environment

### ISSUE THREAD

**Issue by stebogit** [Turfjs/turf#1868]
Hello folks,
whenever someone creates an issue for Turf I always found annoying needing to run the posted code example locally to be able to see the result and try to figure out what's going on; I have the impression that in most cases you just need to display the result on the map to understand the problem and I always reach for [geojson.io](http://geojson.io) to render the result of the code. Just so annoying.

Now, I came across [this smart tool](https://try-carbon.herokuapp.com/) to help [Carbon](https://carbon.nesbot.com/) users and developers interaction when a [new issue](https://github.com/briannesbitt/Carbon/issues/new) is created (their guided required procedure is excellent, foolproof!) and I thought it would be a very powerful tool for Turf as well.

So recently I have been working on an initial implementation of [something similar for Turf](https://turf-sandbox.herokuapp.com/):

<img width="1035" alt="Screen Shot 2020-03-22 at 9 36 18 AM" src="https://user-images.githubusercontent.com/12717225/77258146-f0f6e400-6c35-11ea-8f56-1b474b95abb5.png">

What do you guys think?
Would you like the idea of making this a standard tool for helping people creating new issues? There are still few thing to do of course (the main one being how to select and load the Turf package(s) into the editor), but I wanted to have your feedback at this point before going further.

**Comment by rowanwins** [comment]
Hey @stebogit 

There have been a few things like this over the years
https://dropchop.io/
https://github.com/tcql/turf-playground

And I feel like there was another but I can't remember...

I think it would be kinda handy - particularly if people could generate share links which enabled them to drop a link in a ticket for us to view. It would certainly shortcut a few jsfiddle requirements.

If there is anything you need help thinking about let me know  but seems like you've already made a reasonable start :)

Cheers
Rowan

**Comment by stebogit** [comment]
Thanks @rowanwins I also believe it would make it easier for everybody.

Now, do we only have the minified version of the entire library available [via cdn](https://github.com/Turfjs/turf#in-browser)?
It would be useful to have every package available for the browser and it would also make easy to import them in the sandbox project for specific tests/examples.

@mfedderly since you are working on the release process, do you think it would be possible to set up a cdn version of each package, updated on every library release?

**Comment by mfedderly** [comment]
This looks really cool!

Seems like we don't have to do anything special to get things onto that CDN. At least @turf/turf should become available as soon as we publish to NPM (with the minified version). As for the other packages, we could change the build to also make browser-ready versions of those as well. Maybe spin off a ticket if you think we need the sub-packages to be available.

**Comment by stebogit** [comment]
Thank @mfedderly 

If the library version is consistent for every package, and all packages are going to be correctly included in it, I can certainly simply load the single `@turf/turf` minified file.

I think though it would be nice allow users to load previous versions as well (like with a dropdown menu) so they can reference whatever version they are actually using.
Would it be possible to generate (if not already available) a minified file for all the releases published from v5 onward?

**Comment by mfedderly** [comment]
We're working on releasing 6.2.0-alpha.0, from 6.2.0 on they should all have the minified @turf/turf bundle. I think there might be a few things missing in that package that exist in a sub-package, but that's probably a bug not a feature. I assume that some of the previous releases have it set up correctly already so some of them might work.

I'd really rather that people be using the latest version of things once we get the releases shipping again, instead of trying to reproduce/correct bugs in older versions.

**Comment by stebogit** [comment]
That could be actually not a better idea, so people might be encouraged to adopt the newest version.
However, some times you can't really update the library so easily, if there are breaking changes for instance.
Are we deciding (or have already decided and I did not realize...) we will never release patches for older versions?

**Comment by mfedderly** [comment]
I'm not saying that we'll never patch an old version, but right now the team is small enough that maintaining two major versions at once is likely going to be really hard. We're doing a lot of work on the master branch just to bring it back up to date and make it easier to release.

**Comment by stebogit** [comment]
Any idea how I could get the value of the latest published version though?
I currently [take it from the cdn url](https://github.com/stebogit/turf-sandbox/blob/master/client/src/components/withTurf.js#L22) when loading the library, which feels a bit hacky to me.

**Comment by rowanwins** [comment]
In theory you could use the `npms.io` API
https://api-docs.npms.io/#api-Package-GetPackageInfo

But I'm not sure how it works for Turf with the `@turf/turf` - it doesn't seem happy :(
https://api.npms.io/v2/package/%40turf/turf

**Comment by rowanwins** [comment]
Ahhh 
https://api.npms.io/v2/package/%40turf%2Fturf

**Comment by stebogit** [comment]
@rowanwins @mfedderly I added support to Gist for saving and loading snippet, what do you think?

https://turf-sandbox.netlify.app

**Comment by rowanwins** [comment]
Neat! 

I wonder how feasible the return geojson for rendering will be for more complex scenarios?

**Comment by stebogit** [comment]
What could be an issue or not possible to render on a map? I assume Leaflet should be able to render any valid GeoJSON, shouldn't it?

**Comment by rowanwins** [comment]
Yeah not so much a rendering problem for leafet, it might just be fiddily to return a single feature collection showing the various things you want to show, eg an input and an output.

**Comment by stebogit** [comment]
I honestly can't think of an alternative.
I'm open to ideas or suggestions

**Comment by rowanwins** [comment]
Yeah not sure I've got any :) 
Roll with what you've got and perhaps it's not a major issue anyway - there's only one way to find out and that's to use it
